go/vt/vtadmin/api_test.go

```diff
@@ -4942,6 +4967,10 @@ func TestServeHTTP(t *testing.T) {
 
 			api.ServeHTTP(w, req)
 
+			if tt.repeat {
```

Comment:
[two hops this time!!](https://www.youtube.com/watch?v=wZv62ShoStY)

Reply:
easiest way to do it 😅  😅